packages/server/src/core/router.ts

```diff
@@ -209,8 +212,12 @@ export type CreateRouterInner<
  * @internal
  */
 export function createRouterFactory<TConfig extends RootConfig>(
-  defaults?: RouterDefaultOptions<TConfig['ctx']>,
+  defaults: RouterDefaultOptions<TConfig['ctx']>,
 ) {
+  const transformer = defaults?.transformer ?? defaultTransformer;
+  const errorFormatter = defaults?.errorFormatter ?? defaultFormatter;
```

Comment:
The optional chain operator is not needed here, since the `?` flag was removed from `defaults`.

Reply:
However, is it really "safe" to make it non-optional?